sdv/tabular/base.py

```diff
@@ -8,11 +8,32 @@
 import numpy as np
 import pandas as pd
 
-from sdv.errors import ConstraintsNotMetError
 from sdv.metadata import Table
 
 LOGGER = logging.getLogger(__name__)
 COND_IDX = str(uuid.uuid4())
+FIXED_SEED = 21321
+
+
+def validate_sample_args(function):
```

Comment:
Using a decorator for this seems overkill, since this is being used only in one place. On the other hand, using a decorator as it is done here would shadow the builtin `help(model.sample)` (or `model.sample?` in ipython) functionality by showing the `wrapper` docstring (so, nothing) help instead of the actual `sample` docstring.

I think the right way to do this is to keep the argument in the `sample` method for some time, stating that the argument is deprecated in its description, and validate the value right there, inside the `sample` method.

Reply:
I see. Sure I can change it to keeping the argument in sample.